build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -xe
+
+IMAGE_TAG=1.0.0
```

Comment:
IMAGE_TAG=1.4.1-alpine ?
```
Step 3/14 : FROM cloudflare-cfssl:1.4.1-alpine
pull access denied for cloudflare-cfssl, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Reply:
@ymorimo 1.0.0 is for scalar-cfssl, right ?
So, it's either we need two variables for cloudflare-cfssl and scalar-cfssl or we use the same version (1.4.1) for both.

Reply:
@feeblefakie @tei-k Thank you for pointing it out. Yes, 1.0.0 is for `scalar-cfssl`. (I used the independent number because we need to control the version differently from cfssl it depends on.)
I was confused about the two versions and variables. This `build.sh` can't build the image correctly. 😨

Reply:
@feeblefakie @tei-k Just updated the scripts. In addition to fixing the problem of variables, I made Dockerfile read the base image name (`cloudflare-cfssl`) from an argument. And it should be considered temporary so it is removed once the build process is done.